i18n/translations/en.json

```diff
@@ -768,7 +769,7 @@
 	"user.changeLanguage": "Change language",
 	"user.changeName": "Rename this user",
 	"user.changePassword": "Change password",
-	"user.changePassword.current": "Your current password",
+	"user.changePassword.current": "Your own password",
```

Comment:
I think we need two different variants here depending on whether editing one own password or another user. For editing another user, this change is great. But when chasing your own password, "current" is actually the better phrasing.